crates/router/src/connector/checkout/transformers.rs

```diff
@@ -581,17 +582,29 @@ impl TryFrom<&types::PaymentsCaptureRouterData> for PaymentCaptureRequest {
         let connector_auth = &item.connector_auth_type;
         let auth_type: CheckoutAuthType = connector_auth.try_into()?;
         let processing_channel_id = auth_type.processing_channel_id;
+        let capture_type = if item.request.multiple_capture_data.is_some() {
```

Comment:
Create a util method is_multiple_capture() -> bool for types::PaymentsCaptureRouterData in `crates/router/src/connector/utils.rs`.